<launch>

    <node pkg="gazebo_ros" name="gazebo_launch" type="gzserver" args="/home/jon/catkin_ws/src/finderbot/FinderBot_Gazebo/simple_maze.world"/>

    <node pkg="finderbot" name="exploration" type="path_execution_node" output="screen" />

    <node pkg="finderbot" name="global_mapping" type="global_map_node" output="screen" />

</launch>